services/connectivity/src/service/connectivity.js

```diff
@@ -48,10 +90,16 @@ module.exports = () => {
      * @returns {Object} Geojson object with the geometry
      */
     getPAConnectivityLayers: async (areaType, areaId, paNumber) => {
-      if (paNumber === 'undefined' || paNumber <= 0) {
-        return connectivityPALayersAll;
+      try {
+        return await connectivityPersistence.findPAConnectivityLayers(areaType, areaId, paNumber);
+      } catch (e) {
+        const error = {
+          code: 500,
+          stack: e.stack,
+          message: 'Error retrieving PAConnectivityLayers data',
+        };
+        throw error;
```

Comment:
You should only catch errors in the service layer when you make requests to third party services and you want to hide the error message from your users

Reply:
Done